tests/test_proxy_functional.py

```diff
@@ -118,6 +119,7 @@ async def test_secure_https_proxy_absolute_path(
 
     await sess.close()
     await conn.close()
+    await asyncio.sleep(0.1)
```

Comment:
`0.1` seems a bit brittle. Does this need one iteration of the event loop or is it waiting for a specific timer?
```suggestion
    await asyncio.sleep(0)
```

Reply:
I'm not sure. I was experimenting w/o really debugging in many of the cases. And sometimes `0` wasn't helping… I'll need to take a look again, I suppose.